templates/partials/ensure-no-defaults.dockerfile.j2

```diff
@@ -0,0 +1,6 @@
+{# This partial file is used to ensure there are no packages from the defaults conda channel #}
+
+{# Check for defaults packages in rapids env #}
+RUN source activate rapids \
```

Comment:
Should we also check the `base` environment as well?

Reply:
Ah, that makes sense.

Reply:
Right Conda's. Sorry for the confusion